package.json

```diff
@@ -1,25 +1,28 @@
 {
+  "name": "ui",
+  "version": "0.0.1",
+  "author": {
+    "name": "Riipen",
+    "email": "admin@riipen.com",
+    "url": "https://riipen.com"
+  },
+  "license": "MIT",
+  "repository": "https://github.com/riipen/ui",
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --hoist",
     "build": "lerna run build",
     "ci:audit": "lerna run ci:audit",
     "format": "lerna run format",
-    "lint": "lerna run lint"
+    "lint": "lerna run lint",
+    "start": "lerna --scope riipen-ui-docs run start --stream"
```

Comment:
NOw you can start the docs site from the root of the repo.

Reply:
ah thats really nice actually